src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+
+import classNames from 'classnames';
+import { getSearchWith } from '../../utils/searchHelper';
+import { Gender } from '../../types/Gender';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const CENTURIES_LINKS = ['16', '17', '18', '19', '20'];
+  const SEX_FILTER_DATA = {
+    All: null,
+    Male: 'm',
+    Female: 'f',
+  };
```

Comment:
Static constants should be kept outside the component so that they are not created each time the component is rendered.